.github/workflows/spark-rapids-jni.yaml

```diff
@@ -19,4 +19,4 @@ jobs:
       - name: "Build spark-rapids-jni"
         run: |
           mkdir target
-          CMAKE_CUDA_ARCHITECTURES=90 LIBCUDF_DEPENDENCY_MODE=latest USE_GDS=on scl enable gcc-toolset-11 build/buildcpp.sh
+          CMAKE_CUDA_ARCHITECTURES=90 LIBCUDF_DEPENDENCY_MODE=latest USE_GDS=on scl enable gcc-toolset-14 build/buildcpp.sh
```

Comment:
the CI leverage the same image for both 25.08 and 25.10, please help update https://github.com/NVIDIA/spark-rapids-jni/pull/3568#discussion_r2234213283 to include multiple versions, it would be easy to support both, thanks